std/experimental/allocator/building_blocks/bitmapped_block.d

```diff
@@ -806,7 +809,7 @@ struct BitmappedBlock(size_t theBlockSize, uint theAlignment = platformAlignment
     uint blockSize = 64;
     auto a = BitmappedBlock!(chooseAtRuntime, 64)(cast(ubyte[])(r.allocateAll()), blockSize);
     static assert(hasMember!(InSituRegion!(10_240, 64), "allocateAll"));
-    const b = a.allocate(100);
+    const b = (() pure nothrow @safe @nogc => a.allocate(100))();
```

Comment:
Why do we need this lambda? The `unittest` is still @system

Reply:
It looks like it's so if `allocate` stops being `@safe` in the future this unittest will fail.

Reply:
Sorry for being so unclear, I wanted to ask: "Why not make the unittest @safe?"
(we/I periodically grep for "@system unittest" for this list - https://github.com/dlang/phobos/projects/5)

Reply:
All those attributed lambdas were added with each method that got attributed. We will be able to do a cleanup (i.e. add the attributes to the unittest itself) once all the methods have their attributes. I don't like the looks either :smile: